src/constants/index.js

```diff
@@ -23,6 +23,29 @@ export const CURRENCIES = [
         symbol: '₿',
         decimal: '.',
         thousand: ','
+    },
+    {
+        name: '',
+        symbol: '',
+        decimal: '.',
+        thousand: ','
+    }
+]
+export const EXPECTED_BUDGET_TIMEFRAME_OPTIONS = [
```

Comment:
Maybe we could have a `value/label` pattern here which is common where the `value` is the string to be stored in the DB that needs to never change, and then a `label` would be the human-readable string that could be changed to whatever in the future without having to worry about data integrity in the database (good for multiple language support as well)